### PR TITLE
Load in routes *after* mutator plugins

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @btholt or @danehansen

### CODE REVIEW

File: src/views/pages/index.jsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ class IndexPage extends React.Component {
       <main>
         { loading }
         <TopSubnav user={ user } sort={ sort } list='comments' baseUrl={ this.props.url }/>
-        <div className='container listing-container'>
+        <div className='container listing-container' ref='listings'>
```
Comment (btholt) [inline src/views/pages/index.jsx]: Is this ref used somewhere? I don't see it.

**Comment by btholt** [comment]
:nail_care:

File: src/views/pages/index.jsx (hunk 1 above)
Comment (ajacksified) [inline src/views/pages/index.jsx]: It's used in the `ads` plugin.

Comment (btholt) [inline src/views/pages/index.jsx]: We can use refs. I just thought this one was unused. LGTM

**Comment by btholt** [comment]
:fish:

**Pull request merged**